stingray/utils.py

```diff
@@ -346,37 +359,144 @@ def get_random_state(random_state=None):
     return random_state
 
 
-def baseline_als(y, lam, p, niter=10):
+def _offset(x, off):
+    """An offset."""
+    return off
+
+
+def offset_fit(x, y, offset_start=0):
+    """Fit a constant offset to the data.
+
+    Parameters
+    ----------
+    x : array-like
+    y : array-like
+    offset_start : float
+        Constant offset, initial value
+
+    Returns
+    -------
+    offset : float
+        Fitted offset
+    """
+    from scipy.optimize import curve_fit
+    par, _ = curve_fit(_offset, x, y, [offset_start],
+                       maxfev=6000)
+    return par[0]
+
+
+def _als(y, lam, p, niter=10):
     """Baseline Correction with Asymmetric Least Squares Smoothing.
 
     Modifications to the routine from Eilers & Boelens 2005
-    https://www.researchgate.net/publication/228961729_Technical_Report_Baseline_Correction_with_Asymmetric_Least_Squares_Smoothing
+    https://www.researchgate.net/publication/
+        228961729_Technical_Report_Baseline_Correction_with_
```

Comment:
I guess I would prefer to keep the URL intact rather than breaking it over several lines, but it's not a big deal of course.